src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends FigureTwo implements AreaOfFigure, InfoFigure {
+    //прямоугольник
+
+    public Rectangle(String color,double firsSide, double secondSide) {
```

Comment:
There is a typo in the parameter name `firsSide`. It should be `firstSide` to maintain clarity and consistency in naming conventions.